README.md

```diff
@@ -10,9 +10,9 @@
 Build | Status
 -- | --
 Test coverage | [![codecov](https://codecov.io/gh/gazebosim/gz-tools/branch/main/graph/badge.svg)](https://codecov.io/gh/gazebosim/gz-tools/branch/main)
-Ubuntu Focal  | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_tools-ci-main-focal-amd64)](https://build.osrfoundation.org/job/ignition_tools-ci-main-focal-amd64)
-Homebrew      | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ignition_tools-ci-main-homebrew-amd64)](https://build.osrfoundation.org/job/ignition_tools-ci-main-homebrew-amd64)
-Windows       | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=ign_tools-ci-clowin)](https://build.osrfoundation.org/job/ign_tools-ci-clowin)
+Ubuntu Jammy  | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gz_tools-ci-gz-main-jammy-amd64)](https://build.osrfoundation.org/job/gz_tools-ci-gz-main-jammy-amd64)
+Homebrew      | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gz_tools-ci-gz-main-homebrew-amd64)](https://build.osrfoundation.org/job/gz_tools-ci-gz-main-homebrew-amd64)
+Windows       | [![Build Status](https://build.osrfoundation.org/buildStatus/icon?job=gz_tools-ci-clowin)](https://build.osrfoundation.org/job/gz_tools-ci-clowin)
```

Comment:
The urls are not correct? They show up as `not run`. 

As for ubuntu build, we can use Noble:
https://build.osrfoundation.org/job/gz_tools-ci-main-noble-amd64/